grocery_delivery.gemspec

```diff
@@ -14,7 +14,7 @@
 
 Gem::Specification.new do |s|
   s.name = 'grocery_delivery'
-  s.version = '0.0.9'
+  s.version = '0.0.10'
```

Comment:
please don't bump the version in PRs. We'll bump when we're ready to release/

Reply:
Fixed in 5bad3c7.